### PR TITLE
Example thesis.md

### PR DESCRIPTION
**Pull request by oberien**
Blocked on #32

### CODE REVIEW

File: examples/thesis/thesis.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+```config
+lang = "en"
+document_type = "thesis"
+title = "Fancy Title"
+subtitle = "Subtitle (e.g. title in native language)"
+author = "My Name"
+date = "\\today"
+supervisor = "My Supervisor"
+advisor = "My Advisor"
+logo_university = "https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/40px-Markdown-mark.svg.png"
```
Comment (197g) [inline examples/thesis/thesis.md]: Maybe use: https://cdn.pixabay.com/photo/2017/12/17/14/12/bitcoin-3024279_960_720.jpg
See https://pixabay.com/illustrations/bitcoin-cryptocurrency-virtual-3024279/ for license.

File: examples/thesis/thesis.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+logo_faculty = "https://upload.wikimedia.org/wikipedia/commons/thumb/9/92/LaTeX_logo.svg/40px-LaTeX_logo.svg.png"
+university = "Univeristy of Duckburg"
+faculty = "Department of Quak"
+thesis_type = "Bachelor's Thesis in Quak"
+location = "Duckburg"
+disclaimer = "I confirm that this master's thesis is my own work and I have documented all sources and material used."
```
Comment (197g) [inline examples/thesis/thesis.md]: This is supposedly a `Bachelor's Thesis in Quak`, inconsistent with the disclaimer.

**Review by 197g — COMMENTED** [review]
Dependent on #32 but otherwise nice. Well, the disney references are half hearted ;)

File: examples/thesis/thesis.md (hunk 1 above)
Comment (oberien) [reply examples/thesis/thesis.md]: This is horribly fantastic :D

**Pull request merged**